pkg/build/apiserver/registry/buildconfiginstantiate/rest.go

```diff
@@ -206,7 +206,7 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 					}
 				}
 			}
-			glog.V(2).Infof("failed to instantiate: %#v", request)
+			glog.V(2).Infof("failed to instantiate: %#v with error %#v", request, err)
```

Comment:
i don't believe the `#` is adding anything useful here or in the other spots, probably just making the output less readable.

Reply:
(the # on the err format.  the one on the request is useful since it's a struct)

Reply:
so just #v for the err then ?

Reply:
```suggestion
			glog.V(2).Infof("failed to instantiate: %#v with error %v", request, err)
```

Reply:
sorry I meant %v which is what your suggestion does 

I'll just include it with what I move to openshift/ose